src/pages/layer-2.js

```diff
@@ -271,6 +271,11 @@ const Layer2Page = ({ data }) => {
         pathId: "use-layer-2",
         isSecondary: "isSecondary",
       },
+      {
+        content: "How to get onto layer 2",
```

Comment:
We should use a dynamic verb in the CTA here IMO

```suggestion
        content: "Move to layer 2",
```